perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -940,6 +954,24 @@ sub set_report_extras : Private {
         }
     }
 
+    my @extra_fields = $c->model('DB::ReportExtraFields')->for_cobrand($c->cobrand)->for_language($c->stash->{lang_code})->all;
```

Comment:
Can this call setup_report_extra_fields or has that already been done and the stash can be used?